Microsoft.DotNet.Interactive.PowerShell/PowerShellKernel.cs

```diff
@@ -56,6 +58,19 @@ public PowerShellKernel()
                     $"{psJupyterModulePath}{Path.PathSeparator}{psModulePath}");
 
                 RegisterForDisposal(pwsh);
+
+                // Register type accelerators for Plotly.
+                var accelerator = typeof(PSObject).Assembly.GetType("System.Management.Automation.TypeAccelerators");
+                MethodInfo addAccelerator = accelerator.GetMethod("Add", new Type[] { typeof(string), typeof(Type) });
+                foreach (Type type in typeof(Graph).GetNestedTypes())
+                {
+                    addAccelerator.Invoke(null, new object[] { type.Name, type });
+                }
+
+                // Add accelerators that exist in other namespaces.
+                addAccelerator.Invoke(null, new object[] { "Layout", typeof(Layout.Layout) });
+                addAccelerator.Invoke(null, new object[] { "Chart", typeof(Chart) });
```

Comment:
This should be moved out to a static constructor. In case that more than one PowerShell kernel get created in future, this code will run twice and exception may be thrown from the `Add` call due to already existing keys.

Reply:
Also, do we need to add accelerators for all nested types in `Graph`? 2 concerns here:
1. There is a big list of nested types in `Graph`, and it doesn't seem all of them will be really used for the supported scenarios, does it? (e.g. `Graph.X`, `Graph.Stream`, `Graph.Domain`)
2. There is a possibility that a nested type has the same short name as one of our built-in accelerator type, and that will result in an exception thrown from `Add`. so, it's better to only add those types that are needed for the supported plotting scenarios.
    Also, we probably can use `Graph.Scatter` as the key for the accelerator type, namely using the `Graph.` prefix, so there won't be a collision in the keys.

Reply:
> There is a big list of nested types in Graph, and it doesn't seem all of them will be really used for the supported scenarios, does it?

All in [Graph.fs](https://github.com/fslaborg/XPlot/blob/master/src/XPlot.Plotly/Graph.fs) are what we want to support. It looks like there are a few extras. I can craft a skip list.

> Also, we probably can use Graph.Scatter as the key for the accelerator type, namely using the Graph. prefix, so there won't be a collision in the keys.

Good idea. Maybe since we also have `Layout` and `Chart` that aren't in `Graph` we can do `Plotly.` as the prefix?

Reply:
All nested types under `Graph` can be found here: https://fslab.org/XPlot/reference/xplot-plotly-graph.html

Reply:
> Maybe since we also have Layout and Chart that aren't in `Xplot.Plotly.Graph` we can do `Plotly.` as the prefix?

I would say we don't add prefix for those 2.

Reply:
> All nested types under Graph can be found here: https://fslab.org/XPlot/reference/xplot-plotly-graph.html

Right. The current PR exposes all of those. Do we want all of those exposed? Or do we just want a subset?

Reply:
talked offline... I'll expose all with `Graph.` prefix.

Reply:
Done.